opm/simulators/utils/MPIPacker.cpp

```diff
@@ -70,34 +70,38 @@ packSize(const std::string& data, Parallel::MPIComm comm)
     MPI_Pack_size(1, Dune::MPITraits<std::size_t>::getType(), comm, &size);
     int totalSize = size;
     MPI_Pack_size(data.size(), MPI_CHAR, comm, &size);
-    return totalSize + size;
+    return static_cast<std::size_t>(totalSize + size);
```

Comment:
The explicit cast does not hurt, but there should be an implicit one for the original code anyway.